cv/stats.cpp

```diff
@@ -145,7 +144,7 @@ DMZ_INTERNAL void llcv_equalize_hist(const IplImage *srcimg, IplImage *dstimg) {
   {
     sum += hist[i];
     int val = cvRound(sum*scale);
-    lut[i] = CV_CAST_8U(val);
+    lut[i] = val;
```

Comment:
Are you sure it's safe? If I got it right `CV_CAST_8U` clamps the value to [0...255] range
```
#define  CV_CAST_8U(t)  (uchar)(!((t) & ~255) ? (t) : (t) > 0 ? 255 : 0)
```

Reply:
No, I didnt even look at what this was doing, since I tried scanning and everything worked. Have another look, I went with a different solution